packages/lib/gpc/src/gpcCompile.ts

```diff
@@ -387,6 +392,80 @@ function compileProofObject(objInfo: CompilerObjInfo<POD>): ObjectModuleInputs {
   };
 }
 
+function compileProofBoundsChecks<
+  ObjInput extends POD | GPCRevealedObjectClaims
+>(
+  boundsCheckConfig: GPCProofBoundsCheckConfig,
+  entryMap: Map<PODEntryIdentifier, CompilerEntryInfo<ObjInput>>,
+  paramBoundsChecks: number
+): {
+  boundsCheckEntryIndices: CircuitSignal[];
+  boundsCheckMinValues: CircuitSignal[];
+  boundsCheckMaxValues: CircuitSignal[];
+} {
+  // Arrange POD entry identifiers alphabetically
+  const boundsCheckIdOrder = (
+    Object.keys(boundsCheckConfig) as PODEntryIdentifier[]
+  ).sort((id1: PODEntryIdentifier, id2: PODEntryIdentifier): number => {
```

Comment:
I'd suggest splitting out this custom comparator into a function in gpcUtil.
Alternately, I forgot last time to suggest that the entries in the entryMap area already ordered this way, because that's how EntryModules are ordered.  There might be a version of this function you iterate through that to implicitly keep the same order, and use push() to build your output arrays (skipping entries with no bounds).